internal/users/users.go

```diff
@@ -34,13 +37,17 @@ type GroupInfo struct {
 type options struct {
 	groupPath  string
 	gpasswdCmd []string
+	getentCmd  []string
 }
 
 var defaultOptions = options{
 	groupPath:  "/etc/group",
 	gpasswdCmd: []string{"gpasswd"},
+	getentCmd:  []string{"getent", "passwd"},
 }
 
+var defaultOptionsMu sync.RWMutex
```

Comment:
See my other comments about the mutex. As this is only exported for tests, I wouldn’t add it and ensure the tests calling it aren’t run in parallel (as the mutex doesn’t even fix this use case)